### ISSUE TITLE
Unable to run on iOS device

### ISSUE THREAD

**Issue by berlonce** [makepad/makepad#296]
Created new project using Xcode:
org name=testorg
app name=testapp
xcode version=15.0
project format = 14.0
dev machine os = MacOS 14.0 on mac min M1

Then successfully ran xcode generated app on iOS device (iPhone 14 Pro, iOS 17.0.3) with "automatically manage signing" enabled.

Then from makepad directory ran:
cargo makepad ios --org-id=[my team id] --org=testorg --app=testapp run-device -p makepad-example-simple --release
(same team id as listed in the provisioning profile (only one profile exists) located in /Users/[username]/Library/MobileDevice/Provisioning Profiles/

Failed with the following error output (actual team id replace with [my team id]):

Finished release [optimized] target(s) in 0.08s
Got error: Process xcrun in dir "/Users/[username]/Development/makepad" returned error exit code
ERROR: Failed to install the app on the device. (com.apple.dt.CoreDeviceError error 3002.)
         NSURL = file:///Users/[username]/Development/makepad/target/makepad-ios-app/aarch64-apple-ios/release/makepad-example-simple.app/
       ----------------------------------------
       Unable to Install “testapp” (IXUserPresentableErrorDomain error 14.)
         NSLocalizedRecoverySuggestion = Failed to verify code signature of /var/installd/Library/Caches/com.apple.mobile.installd.staging/temp.2Du4fe/extracted/makepad-example-simple.app : 0xe8008018 (The identity used to sign the executable is no longer valid.)
         NSLocalizedFailureReason = This app cannot be installed because its integrity could not be verified.
       ----------------------------------------
       Failed to verify code signature of /var/installd/Library/Caches/com.apple.mobile.installd.staging/temp.2Du4fe/extracted/makepad-example-simple.app : 0xe8008018 (The identity used to sign the executable is no longer valid.) (MIInstallerErrorDomain error 13.)
         LegacyErrorString = ApplicationVerificationFailed
         SourceFileLine = 78
         LibMISErrorNumber = -402620392
         FunctionName = +[MICodeSigningVerifier _validateSignatureAndCopyInfoForURL:withOptions:error:]

13:31:50  Acquired tunnel connection to device.
13:31:50  Enabling developer disk image services.
13:31:50  Acquired usage assertion.
2%... 3%... 4%... 5%... 6%... 7%... 8%... 9%... 10%... 11%... 12%... 13%... 14%... 15%... 16%... 18%... 19%... 20%... 21%... 22%... 23%... 24%... 25%... 26%... 27%... 28%... 30%... 31%... 32%... 33%... 34%... 36%... 37%... 38%... 39%... 40%... 41%... 42%... 43%... 44%... 45%... 46%... 48%... 49%... 50%... 51%... 52%... 53%... 54%... 55%... 56%... 57%... 59%... 60%... 62%... 66%... 68%... 72%... 

[then quit]

**Comment by berlonce** [comment]
Setup another new app (app name=testapp, org name=makepad), with "automatically manage signing" unchecked, and using a development profile (removed all other profiles), I am able to run the xcode generated app on device but when trying this:
cargo makepad ios --org-id=[my team id] --org=makepad --app=testapp run-device -p makepad-example-simple --release
I am getting this error:
Finished release [optimized] target(s) in 0.11s
Got error: Process xcrun in dir "/Users/[username]/Development/makepad" returned error exit code
ERROR: The specified device was not found. (com.apple.dt.CoreDeviceError error 1000.)

**Comment by guofoo** [comment]
Just some checks to see if you have gone through these steps:

Have you [enabled developer mode]( https://www.delasign.com/blog/how-to-turn-on-developer-mode-on-an-iphone/)?

And then in the VPN & Device Management screen, tapped Developer App with your dev. account and allowed it to install apps?

Were you able to first deploy the "stub", "empty" blank app from and using XCode to the device successfully first?  So that the device contains the app+icon, which when ran, shows the "Hello, world!" on screen? 

Once that is done, the "cargo makepad ios" command should be able to proceed and install/run properly.

**Comment by guofoo** [comment]
Closing the issue for now as there's been no activity for 10 months. Please re-open or raise a new issue if this is still a problem.
